### ISSUE TITLE
make: *** [all] Error 255

### ISSUE THREAD

**Issue by krasnovpro** [rogual/neovim-dot-app#303]
How to fix it?

```
~$ b i neovim-dot-app
==> Installing neovim-dot-app from rogual/neovim-dot-app
==> Downloading https://github.com/rogual/neovim-dot-app/archive/0.1.5.tar.gz
Already downloaded: /Users/oleg/Library/Caches/Homebrew/neovim-dot-app-0.1.5.tar.gz
==> make NVIM=/usr/local/opt/neovim/bin/nvim
Last 15 lines from /Users/oleg/Library/Logs/Homebrew/neovim-dot-app/01.make:
2016-10-16 22:55:32 +0300

make
NVIM=/usr/local/opt/neovim/bin/nvim

VIM= NVIM=/usr/local/opt/neovim/bin/nvim scons -Q
Please set VIM to Neovim's $VIM directory.
make: *** [all] Error 255

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/rogual/homebrew-neovim-dot-app/issues

~$
```

**Comment by prog666** [comment]
i handled this by building it manually with
`$ VIM=NVIM=/usr/local/bin/nvim make`

**Comment by bambu** [comment]
This may be a duplicate of #282 which should be fixed with pr #297. You may need to do a `brew install --HEAD` to get the latest.

@prog666 $VIM is supposed to be pointing to the $VIM variable shown when running `nvim --version` not the nvim executable. Good to know that it still works without it being set correctly
